### DISCUSSION TITLE
Batch Video Render?

### DISCUSSION THREAD

**Discussion by MaliceUnarmed** [n00mkrad/flowframes#190] [Q&A]
Is there an option to render multiple video files? Can't select multiple videos and I assume the Browse Folder for Input is for images. 

I am loving this program, I want to go back and re-do all my YT content with this because even with recording at 60fps some scenes just don't feel like it or there is an odd stutter. Using this program for such scenes really smooths all that out.

**Comment by g0ll4m** [comment]
I would also like this, also a way to batch multiple image sequences for animation!

**Comment by n00mkrad** [comment]
Just drop multiple items into the GUI or use the Batch Processing queue button.

**Reply by g0ll4m** [reply]
Is there a way to add the current/same configuration to all items in the queue?  When I add, they all default to 0 FPS...

**Reply by g0ll4m** [reply]
I render massive volumes of shots in animation.  I'm hoping I can add like 50 folders to the Batch Processing window and all items in the queue can be set to the same "Output FPS and Speed" settings at once.

Right now it looks like I have to individually add each folder, set the output speed, then press add current configuration to queue.  I'm not complaining I already love this build and will be using it for a long time.  I'm just hoping this will be possible if not now, in the near future.

**Reply by g0ll4m** [reply]
I just subbed on Patreon and it looks like this was addressed in the newest build!  Typing in the frame rate for each folder is definitily better!
